src/back/deps/svetit/shared/schemas.cpp

```diff
@@ -0,0 +1,147 @@
+#include "schemas.hpp"
+#include "errors.hpp"
+#include "userver/formats/json/serialize.hpp"
+#include "userver/formats/json/value.hpp"
+#include "userver/formats/json/value_builder.hpp"
+#include "userver/server/http/http_method.hpp"
+
+#include <fmt/core.h>
+#include <ranges>
+#include <utility>
+#include <string>
+#include <fmt/format.h>
+#include <boost/algorithm/string.hpp>
+
+#include <userver/server/handlers/http_handler_json_base.hpp>
+#include <userver/utest/using_namespace_userver.hpp>
+#include <userver/fs/blocking/read.hpp>
+#include <userver/utils/text.hpp>
+
+namespace svetit {
+
+formats::json::Value getBodySchema(
+	const formats::json::Value& requestSchema,
+	const std::string& path
+) {
+	if (!requestSchema.HasMember("requestBody"))
+		return {};
+
+	const auto requestBodyParam = requestSchema["requestBody"];
+	const auto requestBodyPath = path + '/' + boost::algorithm::to_lower_copy(requestBodyParam.As<std::string>());
```

Comment:
утилита должна в параметре указывать точный относительный путь к файлу в том числе в правильном регистре
```suggestion
	const auto requestBodyPath = path + '/' + requestBodyParam.As<std::string>();
```

Reply:
Регистр поправил, теперь всё в нижнем всегда. Влил правку в репу утилиты - пожалуйста, опубликуй в pypi новый пакет.

А насчет относительного пути не понял, так как все генерируемые схемы лежат в одной директории. Относительный путь относительно чего нужно указать? Разве что вот так "./project.json" в параметре писать?